.github/workflows/build_doc.yml

```diff
@@ -44,10 +44,15 @@ jobs:
           mkdir build
           cd build
           cmake -DENABLE_DOCS=ON -DENABLE_PYTHON=ON -DCMAKE_BUILD_TYPE=Release ..
+      
+      - name: Build snippets
+        run: |
+          cmake --build . --target ie_docs_snippets -j`nproc`
```

Comment:
we need reorganize snippets build, because currently it builds the whole OpenVINO and then snippets. We can use only OV headers to build static / object library